app/controllers/api/v1/users_controller.rb

```diff
@@ -29,6 +29,51 @@ def update
         end
       end
 
+      def exist
```

Comment:
Can you please add in some [yard documentation](http://www.rubydoc.info/gems/yard/file/docs/Tags.md) on these new methods.  

Specifically outlining:

- what the method does
- what params should look like; what is it expecting
- a `@see` to any relevant link that would help future developers understand the method, if applicable

Here are some examples of that:

- https://github.com/OperationCode/operationcode_backend/blob/master/app/models/user.rb#L62-L70
- https://github.com/OperationCode/operationcode_backend/blob/master/app/models/resource.rb#L8-L11

-
 https://github.com/OperationCode/operationcode_backend/blob/master/app/models/git_hub_statistic.rb#L21-L25

Reply:
done, let me know if you need more / different info though.